SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -2635,55 +2636,28 @@ void USpatialReceiver::QueueAddComponentOpForAsyncLoad(const Worker_AddComponent
 {
 	EntityWaitingForAsyncLoad& AsyncLoadEntity = EntitiesWaitingForAsyncLoad.FindChecked(Op.entity_id);
 
-	// Skip queuing a duplicate AddComponent op.
```

Comment:
Don't we still need to do this if we aren't running with spatial view?

Reply:
This was always incorrect it was just a hack that seemed to not break that was easier than fixing the issue, however I think this has been worked on since then. As this is run over multiple ticks this comes up in both SpatialView and non-SpatialView flows but it makes the Spatial one more awkward unless store ops using an entity delta (which we could do).

However this whole flow seems more complicated than it needs to be. What you probably want is to have a view that contains the values that you check after the async stuff has finished.

Reply:
I spoke to others about what we expected to break if this still doesn't work (as he looked at it before) so we can be on the lookout for that, but it is something that we do need to fix properly if we haven't already.

Reply:
Since SpatialView is supposed to be off by default for now - some users might experience issues with the non-SpatialView code? Is it something safe enough to do?

Reply:
I think this should work now in all cases, although you are right we do need to make sure we don't break existing code. Will check that more thoroughly or just make the entity-delta stuff.
The code that was there before can also cause issues as you can't actually drop those ops. I expect it will often work but I don't think it always will.